verifier/verifier.go

```diff
@@ -252,7 +255,8 @@ func InitVerifier(config *configModel.Configuration, ssiKitClient ssikit.SSIKit)
 		common.JwtTokenSigner{},
 		credentialsConfig,
 		[]VerificationService{
-			&externalSsiKitVerifier,
+			// validation is now done with the trustbloc lib
+			//&externalSsiKitVerifier,
```

Comment:
Not sure if I get that right, so the verifier will not call WaltId anymore right? I thought we are building on waltId for their current (and upcoming) support for did methods apart of web and key? The trustbloc lib will still need to cover the resolver-part to get the key/cert needed for validating the signature or?

Reply:
I removed it, since: 
- currently only web and key are relevant
- ssi-kit is retired and would require us to do a refactoring for using the new verison
- trustbloc is able to resolve web and key
- for new methods(most probably elsi and ebsi), we need to evaluate if its less work to update to the new walt-id solution or extend the trustbloc one